### ISSUE TITLE
Stardump reload error! Dotnet has stopped working.

### ISSUE THREAD

**Issue by SenerDemiral** [Starcounter/Home#394]
Starcounter 2.4.0.5381
Stardump 1.1.1
SQLite 0.0.7

Stardump unload OK

Stardump reload:
```
Configuration settings:
    Verbose: 4
    DatabaseName: tmax14
    FileName: C:\Starcounter\YDK\tMax14.sqlite3
    SkipColumnPrefixes: [__]
    SkipTablePrefixes: []
    SkipTables: []
    UnloadTables: []
    InsertRowsBufferSize: 200
    InsertObjectNoOffset: 0
    ReassignObjectNo: False
    SkipSchema: False
    SchedulerCount: 2
Reload started C:\Starcounter\YDK\tMax14.sqlite3
```
gives "Dotnet has stopped working"
tMax14.sqlite3 file size is ~20MB.

**Comment by miyconst** [comment]
@SenerDemiral could you please check the Windows logs and see if there is a more meaningful exception available.

Does the database dump contain any sensitive information? Would you be willing to share it with me?

**Comment by SenerDemiral** [comment]
Windows Logs -> Application -> Error Log:
```
Faulting application name: StarDump.exe, version: 1.1.0.1179, time stamp: 0x58224b03
Faulting module name: KERNELBASE.dll, version: 10.0.16299.309, time stamp: 0x90a96867
Exception code: 0xe0530002
Fault offset: 0x0000000000014008
Faulting process id: 0x1ae0
Faulting application start time: 0x01d3bc8b7912073c
Faulting application path: C:\Program Files\Starcounter\StarDump\StarDump.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report Id: 57aa487d-ec3c-4f48-bc14-0031b08efbbd
Faulting package full name: 
Faulting package-relative application ID: 
```
Sqlite3 file:
[bodved.zip](https://github.com/Starcounter/Home/files/1816655/bodved.zip)

**Comment by miyconst** [comment]
Thanks you @SenerDemiral! I have reproduced the issue and investigating it.

**Comment by miyconst** [comment]
@SenerDemiral the issue is fixed now, you can try the custom build `2.4.0.5436` or wait for tomorrow's nightly build.

Again, thank you for testing and reporting.
